bluesky_kafka/__init__.py

```diff
@@ -302,15 +409,20 @@ def no_work_during_wait():
         except Exception:
             self.stop()
             raise
+        finally:
+            self.stop()
 
     def stop(self):
+        """
+        Close the underlying consumer.
+        """
         self._consumer.close()
         self.closed = True
 
 
-class BlueskyConsumer:
+class RemoteDispatcher(Dispatcher):
```

Comment:
I though that this was going to inherit from BlueskyConsumer. Is it better to inherit from Dispatcher?

Reply:
It delegates to a BlueskyConsumer but inherits from Dispatcher to get all the `subscribe` machinery.